research/tensorrt/README.md

```diff
@@ -0,0 +1,147 @@
+# Running the TensorFlow Official ResNet with TensorRT
+
+[TensorRT](https://developer.nvidia.com/tensorrt) is NVIDIA's inference
+optimizer for deep learning. Briefly, TensorRT rewrites parts of the
+execution graph to allow for faster prediction times.
+
+Here we provide a sample script that can:
+
+1. Convert a TensorFlow SavedModel to a Frozen Graph.
+2. Load a Frozen Graph for inference.
+3. Time inference loops using the native TensorFlow graph.
+4. Time inference loops using FP32, FP16, or INT8<sup>1</sup> precision modes from TensorRT.
+
+We provide some results below, as well as instructions for running this script.
+
+<sup>1</sup> INT8 mode is a work in progress; please see [INT8 Mode is the Bleeding Edge](#int8-mode-is-the-bleeding-edge) below.
+
+## How to Run This Script
+
+### Step 1: Install Prerequisites
+
+1. [Install TensorFlow.](https://www.tensorflow.org/install/)
+2. [Install TensorRT.](http://docs.nvidia.com/deeplearning/sdk/tensorrt-install-guide/index.html)
+3. We use the [Official version of ResNet image preprocessing](/official/resnet/imagenet_preprocessing.py). Please checkout the Models repository if you haven't
```

Comment:
nit:  Link text is quite long, I would recommend changing "We use the [Official version of ResNet image preprocessing](/official/resnet/imagenet_preprocessing.py)" to "We use image preprocessing functions from the [Official version of Resnet](/official/resnet/imagenet_preprocessing.py)"